deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -81,6 +81,15 @@ spec:
           - get
           - list
           - watch
+          - delete
+        - apiGroups:
+          - machine.openshift.io
+          resources:
+          - machinesets
+          verbs:
+          - list
+          - get
+          - watch
```

Comment:
Why do we need `watch`?

Reply:
[Here](https://github.com/openshift/windows-machine-config-operator/pull/101/files#diff-9e63f7b0791edcb4a53c944552b88970R155) we are allowing for watching  of resources within the `openshift-machine-api`. So if watch is not added we get ` Failed to watch *unstructured.Unstructured: unknown` error

Reply:
This is strange given that we only want to watch Machines in the `openshift-machine-api` namespace.

Reply:
Looked more into the error. Looks like cache informers use `list` and `watch` calls.  https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/cache/internal/informers_map.go#L207